hazelcast/src/test/java/com/hazelcast/multimap/MultiMapContainerStatisticsTest.java

```diff
@@ -143,6 +157,33 @@ public void testMultiMapContainerStats() {
         assertNewLastAccessTime();
         assertNewLastUpdateTime();
 
+        Map<String, Collection<? extends String>> expectedMultiMap1 = new HashMap<>();
+        // a successful putAll(Map) operation on a Hash backed mmap updates the lastAccessTime and lastUpdateTime
+        expectedMultiMap1.put(key, new ArrayList<>(Arrays.asList("value", "value", "value")));
+        multiMap.putAllAsync(expectedMultiMap1);
+        assertNewLastAccessTime();
+        assertNewLastUpdateTime();
+        // a successful clear operation updates the lastAccessTime and the lastUpdateTime
+        sleepMillis(10);
+        multiMap.clear();
+        assertNewLastAccessTime();
+        assertNewLastUpdateTime();
+
+        // a successful putAll(K,V) operation on a Hash backed mmap updates the lastAccessTime and lastUpdateTime
+        multiMap.putAllAsync(key, expectedMultiMap1.get(key));
```

Comment:
Two things might go wrong here:
1. this is an async call; it is possible that the operation has not yet been completed by the time `assertNewLastAccessTime` below is executed.
2. you miss a `sleepMillis()` statement before this one (similarly to previous code blocks in the same test) to ensure that `putAllAsync` is not executed in the same millisecond as the previous operation.


```suggestion
        sleepMillis(10);
        multiMap.putAllAsync(key, expectedMultiMap1.get(key)).toCompletableFuture().join();
```